go.mod

```diff
@@ -9,9 +9,9 @@ require (
 	github.com/bgentry/speakeasy v0.1.0
 	github.com/btcsuite/btcd v0.21.0-beta
 	github.com/btcsuite/btcutil v1.0.2
-	github.com/confio/ics23/go v0.0.0-20200817220745-f173e6211efb
+	github.com/confio/ics23/go v0.6.3
```

Comment:
can you update the proto-deps on the Makefile too? 🙏 

```
CONFIO_URL 		 = https://raw.githubusercontent.com/confio/ics23/v0.6.2
```

Reply:
@fedekunze thanks for the catch. This is updated- can you mark as approved?